### DISCUSSION TITLE
How to add a dependency to the Windows App SDK Framework package?

### DISCUSSION THREAD

**Discussion by dhoehna** [microsoft/WindowsAppSDK#1278] [Q&A] [answered]
I'm looking through the documentation about the framework package.

The FAQ only says

> Apps just need to reference the framework package in their
manifest.

How exactly does one do that?  I can't find any help on my favorite search engine when I search for "Framework package."  Nothing from Microsoft at least.

**Comment by riverar** [comment, marked as answer]
Framework package references
https://github.com/microsoft/WindowsAppSDK/blob/main/docs/faq.md#what-is-a-framework-package
https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/framework-packages/framework-packages-overview

Believe that FAQ text is referring to the addition of `<PackageDependency Name="Microsoft.WindowsAppSDK.1.0-experimental1" MinVersion="0.218.840.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />` to your appmanifest, although that's not really the workflow devs will follow. (They will instead add a NuGet package reference and the machine will churn for a bit / do that work for you via packaging project.)

**Reply by dhoehna** [reply]
oooohhhh.  There is a Package Dependency node in the appx manifest.  That also does not show up in my favorite search engine.

I wish I would have seen that link about the "framework Packages Overview" earlier.

> <PackageDependency Name="Microsoft.WindowsAppSDK.1.0-experimental1" MinVersion="0.218.840.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />

should be in the FAQ so other people won't ask the same question as me.
